aiida/orm/implementation/django/groups.py

```diff
@@ -159,12 +162,12 @@ def __next__(self):
             def next(self):
                 return next(self.generator)
 
-        return Iterator(self._dbmodel.dbnodes.all())
```

Comment:
Better to replace this with `self._dbmodel.dbnodes.all().iterator()` for lazy loading also while looping with __iter__ and __next__, see
see also here http://blog.etianen.com/blog/2013/06/08/django-querysets/